core-db/src/main/java/io/novafoundation/nova/core_db/model/TinderGovBasketItemLocal.kt

```diff
@@ -0,0 +1,23 @@
+package io.novafoundation.nova.core_db.model
+
+import androidx.room.Entity
+import io.novafoundation.nova.core_db.model.common.ConvictionLocal
+import java.math.BigInteger
+
+@Entity(
+    tableName = "tinder_gov_basket",
+    primaryKeys = ["referendumId", "metaId", "chainId"]
```

Comment:
Should we add foreign keys for metaId and chainId for this table ot be automatically cleaned up upon removal of a meta account or a chain?